build/source/engine/canopySnow.f90

```diff
@@ -246,6 +238,7 @@ subroutine canopySnow(&
  ! *****
  ! update mass of ice on the canopy (kg m-2)
  scalarCanopyIce = scalarCanopyIceIter
+ !write(*,*) scalarCanopySnowUnloading !, scalarThroughfallSnow, flux, delS
```

Comment:
remove debug statements - same for the following lines

Reply:
Done.